src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Colors.values().length);
+        Colors randomColors = Colors.values()[index];
+        return randomColors.toString();
```

Comment:
Use `name()` instead of `toString()` to get the string representation of enum constants. This ensures consistency even if `toString()` is overridden.